tests/runtime/servicers/test_global_predict_servicer_impl.py

```diff
@@ -89,6 +86,20 @@ def test_global_predict_works_on_good_inputs(
     assert response == HAPPY_PATH_RESPONSE
 
 
+def test_global_predict_predict_model_direct(
+    sample_inference_service, sample_predict_servicer, sample_task_model_id
+):
+    """Test that the direct predict_model function can be called to bypass the
+    gRPC handler
+    """
+    response = sample_predict_servicer.predict_model(
+        request_name="SampleTaskRequest",
+        model_id=sample_task_model_id,
+        sample_input=HAPPY_PATH_INPUT,
```

Comment:
something looks funky here, this `HAPPY_PATH_INPUT` is a proto message, not a data model object?

Reply:
Oh, we just never verify in the module that the input is a SampleInputType type, whoops!